docs/upgrade/UPGRADE-7.0.0-beta3.md

```diff
@@ -0,0 +1,259 @@
+        - `ProductManualInputPriceFacade::deleteByProduct()`
+        - `ProductManualInputPriceRepository::getByProductAndDomainConfigs()`
+        - `ProductService::setInputPrice()`
+    - interfaces of following (public and protected) methods have changed:
+        - `InputPriceRecalculator::__construct()`
+        - `ProductDataFactory::__construct()`
+        - `ProductCalculatedPricesType::_construct()`
+        - `ProductPriceCalculation::__construct()`
+    - following classes have been removed:
+        - `AdminProductPriceCalculationFacade`
+        - `InvalidPriceCalculationTypeException`
+        - `ProductBasePriceCalculationException`
+        - `ProductInputPriceService`
+    - due to the removal of `Product::$price` and `PricingGroup::$coefficient` you have to fix your tests
+        - we cannot provide exact instruction for fixing tests as we don't know what do you test
+        - please find hints in tests that we fixed during [#595](https://github.com/shopsys/shopsys/pull/595)
+- remove all usages of `\Shopsys\FrameworkBundle\Command\LoadDataFixturesCommand` and `\Shopsys\FrameworkBundle\Component\DataFixture\FixturesLoader` as we no longer support data fixtures in multiple directories ([#568](https://github.com/shopsys/shopsys/pull/568))
+    - use native doctrine fixtures  
+- we moved multidomain data fixtures in namespace `\Shopsys\FrameworkBundle\DataFixtures\DemoMultidomain` to `\Shopsys\FrameworkBundle\DataFixtures\Demo` ([#568](https://github.com/shopsys/shopsys/pull/568))
+    - check for their usage in your code and change the namespace appropriately
+- change calling of `\Shopsys\FrameworkBundle\DataFixtures\ProductDataFixtureReferenceInjector::loadReferences` ([#568](https://github.com/shopsys/shopsys/pull/568))
+    - the last parameter is no longer `bool`, but `integer` - domain ID
+- `Shopsys\FrameworkBundle\Model\Product\ProductFacade::create()` and `Shopsys\FrameworkBundle\Model\Product\ProductFactory` were modified
+    - if you extended the classes in your project, please check out the changes in the framework ones (and the reasons for the changes) in [#581](https://github.com/shopsys/shopsys/pull/581/files)
+- register templates for new FormTypes (DisplayOnlyCustomerType and OrderItemsType) by adding following lines to `app/config/packages/twig.yml` ([#576](https://github.com/shopsys/shopsys/pull/576))
+    ```yaml
+    - '@ShopsysFramework/Admin/Form/orderItems.html.twig'
+    - '@ShopsysFramework/Admin/Form/displayOnlyCustomer.html.twig'
+    ```
+- OrderFormType in administration is now rendered by default ([#576](https://github.com/shopsys/shopsys/pull/576))
+    - to add new field or group is not now necessary to extend twig template, but is sufficient to create OrderFormTypeExtension (see [example on demoshop](https://github.com/shopsys/demoshop/pull/27/commits/c6a54c6592ebab8cba6d86e47985fb31f511ba30#diff-0c583c8b886a9844ffd793f1afa83ec9))
```

Comment:
This is only description of the PR, isn't it? Should it be here?

Reply:
We rewrote this point to make it an instruction.